### PR TITLE
Added many websites

### LINKED ISSUE #5992: [REQ] www.abc.es
Someone reported anonymously: 
  ### Website URL

https://www.abc.es

### What browser are u using?

Edge 115

### Version

1.1.1

### LINKED ISSUE #6123: [REQ]: www.vogelsbergkreis.de
### Website URL

www.vogelsbergkreis.de

### What browser are u using?

Firefox 115.0

### Version

1.1.1

### Notes

The website is not usable when ISDCAC is active. No scrolling or clicking links or buttons

### LINKED ISSUE #8263: [REQ] as.com
Someone reported anonymously: 
  ### Website URL

https://as.com

### What browser are u using?

Chrome 118

### Version

1.1.1

### LINKED ISSUE #10688: [REQ] www.gera.de
Someone reported anonymously: 
  ### Website URL

https://www.gera.de

### What browser are u using?

Firefox 121

### Version

1.1.2

### Issue type

general

### Notes

Seite nicht mehr scrollbar

### LINKED ISSUE #10739: [REQ]: www.jackjones.com
### Website URL

www.jackjones.com

### What browser are u using?

Firefox 123.0

### Version

1.1.2

### Notes

There is a Gray overlay and I can't click on anything on the website unless I disable the extension

### LINKED ISSUE #10766: [REQ] www.dirsyncpro.org
Someone reported anonymously: 
  ### Website URL

https://www.dirsyncpro.org

### What browser are u using?

Firefox 121

### Version

1.1.2

### Issue type

general

### Notes

site is greyed out

### LINKED ISSUE #10790: [REQ] www.rugbycoachweekly.net
Someone reported anonymously: 
  ### Website URL

https://www.rugbycoachweekly.net

### What browser are u using?

Chrome 120

### Version

1.1.4

### Issue type

general

### Notes

Grey overlay/permission blocker.
Autoblock not working on this particular site.

### LINKED ISSUE #10807: [REQ] www.almaviva.it
Someone reported anonymously: 
  ### Website URL

https://www.almaviva.it

### What browser are u using?

Firefox 121

### Version

1.1.2

### Issue type

general

### Notes

Gray overlay

### LINKED ISSUE #10815: [REQ] www.prenatal.nl
Someone reported anonymously: 
  ### Website URL

https://www.prenatal.nl

### What browser are u using?

Chrome 120

### Version

1.1.4

### Issue type

scrollbar

### LINKED ISSUE #11052: [REQ] Site request: https://as.com/
**Website URL(s)**

- https://as.com/

**Browser**
[ X] Firefox Mobile 

**Extension version**
V1.1.4

### PR DESCRIPTION
**Pull request by miranyx**
Fixed https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/6123
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12665
Fixed the error to resolve https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12679
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12520 and https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12215 duplicates because of me.
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12662
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12653
Fixes: https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12661
fixes the error to resolve https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/10739
Fixed https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/10688
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12641
Fixed the grey overlay to resolve https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/10766.
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12643
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12664
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12656
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12658
Resolves https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/11496
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/10807 the page had a grey background on visit.
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/5992
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12649
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12654
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12646
resolves https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/10790
Fixes https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12652
Fixed https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/10815
Fixed https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/6123

Duplicates below:
Resolves https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/12441
Resolves https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/11423
Resolves https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/11160
Resolves https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/8263
Resolves https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/11052
Resolves https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/issues/11496

### CODE REVIEW

File: src/data/js/5_clickHandler.js [hunk 1]
```diff
@@ -8387,6 +8383,55 @@ function getSelector(host) {
...
+    case "prenatal.nl":
+      return 'button.btn:nth-child(3)';
+    case "plus.nl":
+      return '.gtm-cookies-popup-accept-all-btn';
+    case "neckermann.at":
+      return '.cookieSelectOverlayInlineSave > a:nth-child(2)';
+    case "bernard.fr":
+      return 'button.ot-btn:nth-child(2)';
+    case "as.com":
+      return '._5r2nlf4';
```
Comment (obfuscated-loop) [inline src/data/js/5_clickHandler.js:8427]: Shouldn't be a randomly generated ID, use `#redesignCmpWrapper a[href="https://as.com/"]` as it should remain static.

**Review by obfuscated-loop — CHANGES_REQUESTED** [review]
Please fix requests, other than that, looks good to merge and thanks for your help! :+1:

**Comment by obfuscated-loop** [comment]
I have force pushed the fixes to your branch. Thanks again. Merging.

**Review by obfuscated-loop — APPROVED** [review]

**Comment by thibaultmol** [comment]
Are these also not solved? 
image

**Pull request merged**